From 809ac14b16293c5671baddf837446717f663b540 Mon Sep 17 00:00:00 2001
From: Florian Schmaus <flo@geekplace.eu>
Date: Sun, 26 Jan 2020 19:03:03 +0100
Subject: [PATCH] setup.py: tell find_packages() to exclude tests/

It is common practice to not install the tests, hence we exclude them.

Fixes #42.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index aaa1049..25d283f 100755
--- a/setup.py
+++ b/setup.py
@@ -63,7 +63,7 @@ setup(
     license='BSD',
     version='2.3.5',
     url='https://github.com/mirumee/google-i18n-address',
-    packages=find_packages(),
+    packages=find_packages(exclude=["tests"]),
     include_package_data=True,
     install_requires=['requests>=2.7.0'],
     tests_require=['mock', 'pytest-cov', 'pytest'],
-- 
2.24.1

